app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletViewState.kt

```diff
@@ -1,9 +1,11 @@
 package org.p2p.wallet.home.ui.wallet
 
 import org.p2p.wallet.home.ui.wallet.mapper.model.StrigaKycStatusBanner
+import org.p2p.wallet.striga.offramp.models.StrigaOffRampToken
 import org.p2p.wallet.striga.onramp.interactor.StrigaOnRampToken
 
 data class WalletViewState(
```

Comment:
WalletStrigaViewState :))

Reply:
We definitely should rename "Wallet" screen to "Striga"